runbotics/runbotics-desktop/src/action/sharepoint/excel/sharepoint-excel.action-handler.ts

```diff
@@ -41,22 +41,26 @@ export default class SharepointExcelActionHandler extends StatelessActionHandler
             throw new Error(SharePointExcelErrorMessage.getCellIncorrectInput());
         }
 
-        const { value, text, numberFormat } = await this.excelService.getCell({
+        const cellValue = await this.excelService.getCell({
             column: column.toString(),
             row: row.toString()
         });
 
-        if (numberFormat.includes('@') || numberFormat.includes('%')) {
-            return text;
-        }
-
-        return value;
```

Comment:
won't be there a problem with data value?

Reply:
I see you've adapted this solution in getRange function. But this way not only I'll be harder to customize getCells with e.g. checkbox for user to decide if he/she/it wants but also each other function which uses getRange is under the influence of isValueUnclear. I suggest to return from getRange object - both value and text and decide here what should be returned.